### PR TITLE
Override changes

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: recipes-bsp/l-loader/l-loader-poplar_git.bb [hunk 1]
```diff
@@ -18,7 +18,7 @@ SRCREV = "d7b3ac7748d6607c2dd82cf37326b80a2ede6e5b"
 # knowledgeably, it is a hack...
 ###
 SRC_URI = "git://github.com/96boards-poplar/l-loader;branch=latest \
-           http://releases.linaro.org/components/toolchain/binaries/5.3-2016.02/arm-linux-gnueabihf/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf.tar.xz;name=tc \
+           http://releases.linaro.org/components/toolchain/binaries/5.3-2016.02/arm-linux-gnueabihf/gcc-linaro-5.3-2016.02-x86_64:arm-linux-gnueabihf.tar.xz;name=tc \
```
Comment (ndechesne) [inline recipes-bsp/l-loader/l-loader-poplar_git.bb]: not here.

File: recipes-bsp/l-loader/l-loader-poplar_git.bb [hunk 2]
```diff
@@ -29,7 +29,7 @@ do_configure[noexec] = "1"
 
 do_compile() {
     # Use pre-built aarch32 toolchain
-    export PATH=${WORKDIR}/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf/bin:$PATH
+    export PATH=${WORKDIR}/gcc-linaro-5.3-2016.02-x86_64:arm-linux-gnueabihf/bin:$PATH
```
Comment (ndechesne) [inline recipes-bsp/l-loader/l-loader-poplar_git.bb]: not here.

File: recipes-bsp/uefi/edk2-hikey_git.bb [hunk 3]
```diff
@@ -26,19 +26,19 @@ SRC_URI = "git://github.com/96boards-hikey/edk2.git;name=edk2;branch=testing/hik
            git://git.linaro.org/uefi/uefi-tools.git;name=uefitools;destsuffix=git/uefi-tools \
            git://github.com/96boards-hikey/l-loader.git;name=lloader;branch=testing/hikey960_v1.2;destsuffix=git/l-loader \
            git://github.com/96boards-hikey/atf-fastboot.git;name=atffastboot;destsuffix=git/atf-fastboot \
-           http://releases.linaro.org/components/toolchain/binaries/6.4-2017.08/arm-linux-gnueabihf/gcc-linaro-6.4.1-2017.08-x86_64_arm-linux-gnueabihf.tar.xz;name=tc \
+           http://releases.linaro.org/components/toolchain/binaries/6.4-2017.08/arm-linux-gnueabihf/gcc-linaro-6.4.1-2017.08-x86_64:arm-linux-gnueabihf.tar.xz;name=tc \
```
Comment (ndechesne) [inline recipes-bsp/uefi/edk2-hikey_git.bb]: not here

File: recipes-bsp/uefi/edk2-hikey_git.bb [hunk 4]
```diff
@@ -73,9 +73,9 @@ do_compile_prepend() {
     sed -i -e 's:fakeroot ::g' ${S}/l-loader/generate_ptable.sh
 }
 
-fakeroot do_compile_append() {
+fakeroot do_compile:append() {
     # Use pre-built aarch32 toolchain
-    export PATH=${WORKDIR}/gcc-linaro-6.4.1-2017.08-x86_64_arm-linux-gnueabihf/bin:$PATH
+    export PATH=${WORKDIR}/gcc-linaro-6.4.1-2017.08-x86_64:arm-linux-gnueabihf/bin:$PATH
```
Comment (ndechesne) [inline recipes-bsp/uefi/edk2-hikey_git.bb]: not here.

File: recipes-graphics/mesa/mesa_%.bbappend [hunk 5]
```diff
@@ -1,6 +1,6 @@
-PROVIDES_remove = "${@bb.utils.contains("MACHINE_FEATURES", "mali450", "virtual/libgles1 virtual/libgles2 virtual/egl virtual/libgbm", "", d)}"
+PROVIDES:remove = "${@bb.utils.contains("MACHINE_FEATURES", "mali450", "virtual/libgles1 virtual/libgles2 virtual/egl virtual/libgbm", "", d)}"
 
-do_install_append_hikey() {
+do_install:append_hikey() {
```
Comment (ndechesne) [inline recipes-graphics/mesa/mesa_%.bbappend]: missed one override

File: recipes-graphics/wayland/weston_9%.bbappend [hunk 6]
```diff
@@ -1,5 +1,5 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
-SRC_URI_append_poplar = "\
+SRC_URI:append_poplar = "\
```
Comment (ndechesne) [inline recipes-graphics/wayland/weston_9%.bbappend]: missed one.

File: recipes-kernel/linux-firmware/linux-firmware_%.bbappend [hunk 7]
```diff
@@ -2,12 +2,12 @@
 # TIInit_11.8.32.bts is required for bluetooth support but this particular
 # version is not available in the linux-firmware repository.
 #
-SRC_URI_append_hikey = "\
+SRC_URI:append_hikey = "\
```
Comment (ndechesne) [inline recipes-kernel/linux-firmware/linux-firmware_%.bbappend]: missed one.

File: recipes-kernel/linux-firmware/linux-firmware_%.bbappend [hunk 8]
```diff
@@ -2,12 +2,12 @@
...
      https://git.ti.com/ti-bt/service-packs/blobs/raw/c290f8af9e388f37e509ecb111a1b64572b7c225/initscripts/TIInit_11.8.32.bts;name=TIInit_11.8.32 \
 "
 SRC_URI[TIInit_11.8.32.md5sum] = "b1e142773e8ef0537b93895ebe2fcae3"
 SRC_URI[TIInit_11.8.32.sha256sum] = "962322c05857ad6b1fb81467bdfc59c125e04a6a8eaabf7f24b742ddd68c3bfa"
 
-do_install_append_hikey() {
+do_install:append_hikey() {
```
Comment (ndechesne) [inline recipes-kernel/linux-firmware/linux-firmware_%.bbappend]: missed.

File: recipes-kernel/linux/linux-96boards_4.4.bb [hunk 9]
```diff
@@ -9,7 +9,7 @@ SRCREV_FORMAT = "kernel"
 SRC_URI = "git://github.com/96boards/linux.git;protocol=https;branch=96b/releases/2016.06;name=kernel \
           "
 
-SRC_URI_append_hikey = " \
+SRC_URI:append_hikey = " \
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-96boards_4.4.bb]: miss.

File: recipes-kernel/linux/linux-96boards_4.4.bb [hunk 10]
```diff
@@ -40,7 +40,7 @@ KERNEL_CONFIG_FRAGMENTS_hikey += "${S}/kernel/configs/mali-450.conf"
 DEPENDS += "openssl-native"
 HOST_EXTRACFLAGS += "-I${STAGING_INCDIR_NATIVE}"
 
-do_unpack_append_hikey() {
+do_unpack:append_hikey() {
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-96boards_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-lsk-rt-test_4.4.bb [hunk 11]
```diff
@@ -23,7 +23,7 @@ SRC_URI = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-lsk-rt-test_4.4.bb]: miss.

File: recipes-kernel/linux/linux-generic-lsk-rt_4.4.bb [hunk 12]
```diff
@@ -23,7 +23,7 @@ SRC_URI = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-lsk-rt_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-lsk-test_4.4.bb [hunk 13]
```diff
@@ -23,7 +23,7 @@ SRC_URI = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-lsk-test_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-lsk_4.4.bb [hunk 14]
```diff
@@ -23,7 +23,7 @@ SRC_URI = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-lsk_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-stable-rt_4.4.bb [hunk 15]
```diff
@@ -23,7 +23,7 @@ SRC_URI = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-stable-rt_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-stable_4.4.bb [hunk 16]
```diff
@@ -21,7 +21,7 @@ SRC_URI = "\
 "
 
 # apply=yes indicates non-essential patches for STiH410-b2260 support
-SRC_URI_append_stih410-b2260 = "\
+SRC_URI:append_stih410-b2260 = "\
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-stable_4.4.bb]: miss

File: recipes-kernel/linux/linux-generic-stable_4.4.bb [hunk 17]
```diff
@@ -68,7 +68,7 @@ SRC_URI_append_stih410-b2260 = "\
 S = "${WORKDIR}/git"
 
 COMPATIBLE_MACHINE = "am57xx-evm|beaglebone|intel-core2-32|intel-corei7-64|juno|stih410-b2260"
-KERNEL_DEVICETREE_remove_juno = "arm/juno-r2.dtb"
+KERNEL_DEVICETREE:remove_juno = "arm/juno-r2.dtb"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-generic-stable_4.4.bb]: miss

File: recipes-kernel/linux/linux-hikey_git.bb [hunk 18]
```diff
@@ -9,7 +9,7 @@ SRCREV_FORMAT = "kernel"
 SRC_URI = "git://github.com/Linaro/rpk.git;protocol=https;branch=rpk-v4.9;name=kernel \
 "
 
-SRC_URI_append_hikey = " \
+SRC_URI:append_hikey = " \
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-hikey_git.bb]: miss

File: recipes-kernel/linux/linux-hikey_git.bb [hunk 19]
```diff
@@ -45,7 +45,7 @@ KERNEL_CONFIG_FRAGMENTS_hikey += "${S}/kernel/configs/mali-450.conf"
 DEPENDS += "openssl-native"
 HOST_EXTRACFLAGS += "-I${STAGING_INCDIR_NATIVE}"
 
-do_unpack_append_hikey() {
+do_unpack:append_hikey() {
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-hikey_git.bb]: miss

File: recipes-kernel/linux/machine-specific-hooks.inc [hunk 20]
```diff
@@ -17,14 +17,14 @@ KERNEL_EXTRA_ARGS_am57xx-evm += "LOADADDR=${UBOOT_ENTRYPOINT}"
 # Define the kernel device tree for am57xx-evm machine
 # meta-ti is setting more device trees than the ones available in mainline kernel
 KERNEL_DEVICETREE_am57xx-evm = "am57xx-beagle-x15.dtb"
-SRC_URI_append_am57xx-evm = " file://multi_v7_hugepage.config;subdir=git/kernel/configs"
-KERNEL_CONFIG_FRAGMENTS_append_am57xx-evm = " ${S}/kernel/configs/multi_v7_hugepage.config"
+SRC_URI:append_am57xx-evm = " file://multi_v7_hugepage.config;subdir=git/kernel/configs"
+KERNEL_CONFIG_FRAGMENTS:append_am57xx-evm = " ${S}/kernel/configs/multi_v7_hugepage.config"
```
Comment (ndechesne) [inline recipes-kernel/linux/machine-specific-hooks.inc]: miss x2

File: recipes-kernel/linux/machine-specific-hooks.inc [hunk 21]
```diff
@@ -17,14 +17,14 @@ KERNEL_EXTRA_ARGS_am57xx-evm += "LOADADDR=${UBOOT_ENTRYPOINT}"
...
 
 KERNEL_DEVICETREE_beaglebone = "am335x-bone.dtb am335x-boneblack.dtb am335x-bonegreen.dtb"
-SRC_URI_append_beaglebone = " file://multi_v7_hugepage.config;subdir=git/kernel/configs"
-KERNEL_CONFIG_FRAGMENTS_append_beaglebone = " ${S}/kernel/configs/multi_v7_hugepage.config"
+SRC_URI:append_beaglebone = " file://multi_v7_hugepage.config;subdir=git/kernel/configs"
```
Comment (ndechesne) [inline recipes-kernel/linux/machine-specific-hooks.inc]: miss x2

**Comment by kraj** [comment]
thanks @ndechesne for the review. I have fixed all of these cases and uploaded again please take a look.

**Comment by ndechesne** [comment]
thanks for this PR.. at some point someone should look closely at this layer.. i feel it could be cleaned up a lot!

**Pull request merged**